### PR TITLE
Create a request with multi submit actions, builds and diffs

### PR DESCRIPTION
**Pull request by danidoni**
Changed the existing rake task to create a request with multiple submit requests. Those submit requests come from home:Iggy's packages, that push a sample spec to the backend so they build successfully and produce a diff.

For review purposes I added Leap 15.4 under [home:Iggy](https://obs-reviewlab.opensuse.org/danidoni-multi-action-submit-request-with-build-rpmlint-and-diffs-rake-task/project/show/home:Iggy) to kickstart the building process, you can review the sample request [here](https://obs-reviewlab.opensuse.org/danidoni-multi-action-submit-request-with-build-rpmlint-and-diffs-rake-task/request/show/10).

**How to test?**

I recommend starting from a clean backend and database, so

```
 docker-compose stop; docker-compose rm db backend; rake docker:build; docker-compose up
 ```
 
 and run the `test_data` task from the container afterwards
 
 ```
 bundle exec rake dev:test_data:create
 ```
 
The output will give you the request number.
Open that request in your browser and play around with the `Select Action` dropdown, the `Changes` tab, and the `Builds` tab (add repositories first and check after a while).

### CODE REVIEW

**Comment by saraycp** [comment]
It seems this PR overlaps with https://github.com/openSUSE/open-build-service/pull/13731.
Now, the `multiple_actions_request` task covers what  `request_with_multiple_submit_actions_builds_and_diffs` is trying to do.

**Review by krauselukas — COMMENTED** [review]
It works, but the task mentions "builds". They seem missing.

File: src/api/lib/tasks/dev/requests.rake [hunk 1]
```diff
@@ -71,8 +71,9 @@ namespace :dev do
       puts '- Create a couple of repositories in project home:Iggy:branches:home:Admin'
     end
 
-    desc 'Creates several requests with submit actions and diffs'
-    task request_with_multiple_submit_actions_and_diffs: :development_environment do
+    # Creates a request with two actions of the same type: 'submit'.
+    desc 'Creates request with only submit actions and some diffs'
```
Comment (krauselukas) [inline src/api/lib/tasks/dev/requests.rake:75]:
```suggestion
    desc 'Creates a request with only submit actions and some diffs'
```

**Comment by saraycp** [comment]
> It works, but the task mentions "builds". They seem missing.

I add an output message that clarifies that the developer has some manual job to do :smile_cat:

File: src/api/lib/tasks/dev/requests.rake (hunk 1 above)
Comment (saraycp) [reply src/api/lib/tasks/dev/requests.rake]: Pushed

**Review by krauselukas — APPROVED** [review]

**Pull request merged**